### PR TITLE
Minor cleanups of dataset documentation

### PR DESCRIPTION
**Pull request by JamesGuthrie**
## Description

With the latest changes to dataset configuration, some documentation was slightly outdated.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [x] CHANGELOG entry for user-facing changes
- [x] Updated the relevant documentation

### CODE REVIEW

File: docs/dataset.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ Promscale stores some configuration information in the Postgres database which i
...
 
-Expected format is YAML and currently only option that can be set is default chunk interval.
+The expected format is YAML.
 
-Example configuration in config.yaml:
+The YAML values are easier to set when using `config.yaml` instead of the cli flags. This would look as follows:
 
-```
+```yaml
 startup.dataset.config: |
```
Comment (paulfantom) [inline docs/dataset.md:17]: Is this line needed in the YAML file?

**Review by paulfantom — APPROVED** [review]
LGTM. I just have one question asked inline.

File: docs/dataset.md (hunk 1 above)
Comment (JamesGuthrie) [reply docs/dataset.md:17]: it's not needed at all, but it tells the markdown syntax highlighter to highlight as yaml.

Comment (paulfantom) [reply docs/dataset.md:17]: Sorry, I meant line 17: `startup.dataset.config: |`

Comment (JamesGuthrie) [reply docs/dataset.md:17]: I'm not sure that I understand exactly what your question is (I think it has to do with the pipe `|`, but I'm not sure).

Basically all cli flags can be specified in `config.yaml`, and then passed to promscale with `promscale -config config.yaml`. In this case, the `startup.dataset.config` parameter must itself be a yaml-formatted string. The only way to do this is with the `|`.

Maybe you mean, is that config param mandatory? No.

**Review by antekresic — APPROVED** [review]

**Pull request merged**